### PR TITLE
Remove stackpath

### LINKED ISSUE #374: Remove stackpath
- Homepage
- Sponsors
- Infographic
- other?

### PR DESCRIPTION
**Pull request by Drgy**
closes #374, closes #363, closes #364

### CODE REVIEW

**Comment by MartinKolarik** [comment]
In all the places where you removed SP also add Bunny (after Cloudflare, Fastly).

@jimaek we should probably add Bunny also to the map, right?

**Comment by jimaek** [comment]
Yeah lets add them to the map too

**Comment by MartinKolarik** [comment]
@Drgy for the map you update the txt file and then use `node bin/geocode` to update the JSON.

**Comment by MartinKolarik** [comment]
@Drgy while you are at it, update the other CDN locations as well, see #363 and #364 for instructions.

File: data/map.txt [hunk 1]
```diff
@@ -1,3 +1,50 @@
+bunnynet - Adelaide
```
Comment (MartinKolarik) [inline data/map.txt:1]: Bunny

File: src/views/pages/esm.html [hunk 2]
```diff
@@ -69,7 +69,7 @@ <h3 class="subtitle">
 							<h4 class="feature-title">Global Multi-CDN powered by jsDelivr</h4>
 
 							<div class="feature-content">
-								esm.run is powered by jsDelivr and it's sponsors like Cloudflare, Fastly, StackPath, Quantil, and appfleet.
+								esm.run is powered by jsDelivr and it's sponsors like bunnynet, Cloudflare, Fastly, Quantil, and appfleet.
```
Comment (MartinKolarik) [inline src/views/pages/esm.html:72]: wrong order 🙄

Comment (Drgy) [reply src/views/pages/esm.html:72]: The order seems to be alphabetical, is it just coincidence and the order should be Cloudflare, Fastly, Bunnynet, Quantil and appfleet everywhere?

Comment (MartinKolarik) [reply src/views/pages/esm.html:72]: > add Bunny (after Cloudflare, Fastly)

it isn't really AB

**Comment by jimaek** [comment]
Btw we no longer have Madagascar either. Lets remove it?

**Comment by MartinKolarik** [comment]
@Drgy ping https://github.com/jsdelivr/www.jsdelivr.com/pull/375#issuecomment-766428038 and the other comment.

**Pull request merged**